source/customization.rst

```diff
@@ -624,8 +624,8 @@ of 2.0 the existing widgets are:
 - ``xdmod_widget_jobs`` - the XDMoD widget for job information
 
 This feature also allows for administrators to *add* custom widgets.
-Simply drop new files into ``/etc/ood/config/dashboard/views/widgets`` and reference them
-in the configuration.  These partial files can be any format Rails recognizes, notably ``.html`` or
+Simply drop new files into ``/etc/ood/config/apps/dashboard/views/widgets`` or ``ENV["OOD_APP_CONFIG_ROOT”]/views/widgets``
```

Comment:
I think this addition of the `/apps/` directory here is all we want.  All these ENV non-production toggles are advanced settings that should be somewhere else (in some developer documentation we don't currently have).